components/compliance-service/api/tests/04_stats_spec.rb

```diff
@@ -372,63 +384,13 @@ def stats
             "failures" => 21,
             "criticals" => 21,
             "passed" => 22,
-            # COMMENT THIS OUT WHEN WAIVERS API SUPPORT IS ADDED FOR THIS API CALL
-            # "waived" => 3
-          }
-        ]
-      }
-      assert_equal_json_content(expected_data, actual_data)
-
-
-      # Use environment filter with missing env for a specific profile_id
-      actual_data = GRPC stats, :read_profiles, Stats::Query.new(filters: [
-        Stats::ListFilter.new(type: "environment", values: ["Missing In Action"]),
-        Stats::ListFilter.new(type: 'end_time', values: ['2018-03-05T23:59:59Z'])
-      ],
-        type: "controls",
-        id: "09adcbb3b9b3233d5de63cd98a5ba3e155b3aaeb66b5abed379f5fb1ff143988"
-      )
-      expected_data = {}
-      assert_equal_json_content(expected_data, actual_data)
-
-
-      actual_data = GRPC stats, :read_profiles, Stats::Query.new(filters: [
-        Stats::ListFilter.new(type: "environment", values: ["DevSec Prod beta"]),
-        Stats::ListFilter.new(type: 'end_time', values: ['2018-03-05T23:59:59Z'])
-      ],
-        type: "controls",
-        id: "09adcbb3b9b3233d5de63cd98a5ba3e155b3aaeb66b5abed379f5fb1ff143988"
-      )
-      expected_data = {
-        "controlStats" => [
-          {
-            "control" => "nginx-01",
-            "title" => "Running worker process as non-privileged user",
-            "passed" => 1,
-            "impact" => 1
-          },
-          {
-            "control" => "nginx-02",
-            "title" => "Check NGINX config file owner, group and permissions.",
-            "passed" => 1,
-            "impact" => 1
-          },
-          {
-            "control" => "nginx-03",
-            "title" => "Check NGINX three.",
-            "skipped" => 1,
-            "impact" => 1
-          },
-          {
-            "control" => "nginx-04",
-            "title" => "Check NGINX four",
-            "passed" => 1,
-            "impact" => 0.5
+            "waived" => 2
           }
         ]
       }
       assert_equal_json_content(expected_data, actual_data)
 
+      #todo - do we need to handle waiver stats in here too?
```

Comment:
It's the API call used to gather reporting profile details and how each control in the profile performed(failed, passed, skipped or waived)

Needs waivers as per this UI card: https://github.com/chef/automate/issues/2558